packages/core/content-releases/package.json

```diff
@@ -63,6 +63,7 @@
     "axios": "1.6.0",
     "formik": "2.4.0",
     "lodash": "4.17.21",
+    "node-schedule": "2.1.0",
```

Comment:
would it be possible to use the `strapi.cron.add` api? or,  is there any requirement that the api does not cover?

Reply:
Are there any examples or documentation available? 🤔 I'm basing my approach on [what was done for AuditLogs](https://github.com/strapi/strapi/blob/develop/packages/core/admin/ee/server/src/services/audit-logs.ts) since I remember that was the other area where cron jobs were used.

Reply:
I don't see any documentation atm, but I believe you might see some usage of it by looking for "strapi.cron." in the repo.

This should work:
strapi.cron.add({ 'releaseId': { task: callback, options: date } }) 
strapi.crom.remove('releaseId')

Reply:
We should have docs about this 😓 
While your approach is completely fine, I do think it would be better if we use this unified api everywhere.

Reply:
I found [documentation](https://contributor.strapi.io/api/Cron)! I just add something a little detail and also need it to change a type from the CronService but now it's working great! thanks for the advice